skaffold.yaml

```diff
@@ -850,12 +850,7 @@ deploy:
               # TODO(timebertt): drop this after v1.96 has been released
               - bash
               - -ec
-              - |
-                until kubectl get controllerdeployments.v1.core.gardener.cloud >/dev/null; do 
-                  echo "Waiting for controllerdeployments.v1.core.gardener.cloud to be available..."
-                  sleep 5
-                done
-                echo "controllerdeployments.v1.core.gardener.cloud available"
+              - "until kubectl get controllerdeployments.v1.core.gardener.cloud >/dev/null; do \n  echo \"Waiting for controllerdeployments.v1.core.gardener.cloud to be available...\"\n  sleep 5\ndone\necho \"controllerdeployments.v1.core.gardener.cloud available\"\n"
```

Comment:
Why did you change this? 👀

Reply:
I had to run https://github.com/gardener/gardener/blob/master/hack/update-skaffold-deps.sh and it changed the file.